### ISSUE TITLE
Integration with reportportal.io

### ISSUE THREAD

**Issue by neo81-83** [prashant-ramcharan/courgette-jvm#124]
Hello Prashant,

You must have heard of report portal (AI-powered Test Automation Dashboard) which could be very easily integrated with various Selenium frameworks. This dashboard is becoming very popular and it would be great for your solution if you could find a way to get it integrated.

https://reportportal.io/

-W

**Comment by prashant-ramcharan** [comment]
Hello,

I've look into this and the report can be uploaded to report portal after the run is complete.

Will try to fit it in the next release - Courgette 4

Prashant

**Comment by prashant-ramcharan** [comment]
I might roll out a Courgette 4.0.0-SNAPSHOT version with the latest Cucumber version 5.0.0-SNAPSHOT including the report portal integration.

I cannot provide an exact date.. but I've just completed the report integration in Courgette.

**Comment by prashant-ramcharan** [comment]
Note: 

The results will only be published to the report portal server once the Courgette test run is complete. Unfortunately results cannot be published in real time  given the architecture of the test runner.

**Comment by prashant-ramcharan** [comment]
@neo81-83 This is now released!

https://github.com/prashant-ramcharan/courgette-jvm#report-portal-integration

Please use Courgette version 4.0.0-snapshot

_Note: this version also upgrades the Cucumber version to 5.0.0-RC1_

**Comment by neo81-83** [comment]
Thanks @prashant-ramcharan for the quick turnaround, I will check it out and let you know.

**Comment by prashant-ramcharan** [comment]
I've refactored some code and released 4.0.1-snapshot.

Will close this issue now so please open a new issue if you find any :)
